db/types/datetime.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.dialects.postgresql.base import TIME as SA_TIME
+
+from db.types import base
+
+TIME_ZONE_DB_TYPE = base.PostgresType.TIME_WITH_TIME_ZONE.value
+WITHOUT_TIME_ZONE_DB_TYPE = base.PostgresType.TIME_WITHOUT_TIME_ZONE.value
+
+
+class TIME_WITHOUT_TIME_ZONE(SA_TIME):
```

Comment:
Converting the class to a string still returns `Time` rather than `TIME WITHOUT TIME ZONE`. The `str` representation method should be overridden to return the appropriate value